### PR TITLE
UI polishing

### PR DESCRIPTION
**Pull request by ignapas**
## What do these changes do?

* Add a new always visible feedback button on the left of the user menu. Remove the old one.
* Added an info message when the email for a new account has successfully been confirmed.
* Flash messages now show for a longer time.
* Flash messages are now a bit wider.
* Changed background of the navigation bar, and the style of the user account button.
* Catalogue -> Catalog (american stylo)
* Filepicker now opens inside a window
* Styles for different windows unified (filepicker, service catalog, service info...) Added a standard border of 1 px

## Screenshots
Screenshot 2019-06-11 at 09 27 28
Screenshot 2019-06-11 at 09 28 36

## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

**Comment by pcrespov** [comment]
@ignapas Is report issue button always going to be visible? I mean, what happens when you have the inputs bar? Isn't it going to be over the list items?? Wouldn't it be better to put it where it does not interfere e.g. next to the user button?

**Comment by odeimaiz** [comment]
I would also put the "Create issue" button next to the user button. I find it too intrusive.
Also, what about having two different options with their respective labels:
- "give us feedback" for regular users
- "create issue" for testers

**Comment by ignapas** [comment]
> I would also put the "Create issue" button next to the user button. I find it too intrusive.
> Also, what about having two different options with their respective labels:
> 
> * "give us feedback" for regular users
> * "create issue" for testers

I like it I like it. Will do it. Regarding the two options, do we have a give us feedback feature?

**Comment by odeimaiz** [comment]
"Catalogue -> Catalog (american stylo)"
Why?

File: services/web/client/source/class/qxapp/auth/LoginPage.js [hunk 1]
```diff
@@ -72,6 +72,8 @@ qx.Class.define("qxapp.auth.LoginPage", {
       } else if (urlFragment.nav[0] === "reset-password") {
         pages.setSelection([reset]);
       }
+    } else if (urlFragment.params && urlFragment.params.registered) {
+      qxapp.component.message.FlashMessenger.getInstance().logAs(this.tr("Your account have been created.<br>You can now use your credentials to login."));
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/auth/LoginPage.js]: have -> has

File: services/web/client/source/class/qxapp/desktop/StudyEditor.js [hunk 2]
```diff
@@ -239,14 +255,6 @@ qx.Class.define("qxapp.desktop.StudyEditor", {
           this.__nodeView.buildLayout();
           if (node.isInKey("file-picker")) {
             widget = new qxapp.file.FilePicker(node, this.getStudy().getUuid());
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/desktop/StudyEditor.js]: Is this part ever reached?

File: services/web/client/source/class/qxapp/auth/LoginPage.js (hunk 1 above)
Comment (ignapas) [reply services/web/client/source/class/qxapp/auth/LoginPage.js]: right, thanks

File: services/web/client/source/class/qxapp/desktop/StudyEditor.js (hunk 2 above)
Comment (ignapas) [reply services/web/client/source/class/qxapp/desktop/StudyEditor.js]: you mean the part where the filepicker is assigned to widget, or the else ``widget = this.__nodeView``
Both parts should be reached, when it is not the root node and the node is not a container node, that is most of the times, right?

Comment (odeimaiz) [reply services/web/client/source/class/qxapp/desktop/StudyEditor.js]: My bad I missed up line numbers

**Comment by ignapas** [comment]
> "Catalogue -> Catalog (american stylo)"
> Why?

Before it was ServicesCatalogue. I wanted to change it to ServiceCatalogue. Then I thought Catalogue was misspelled as well, it sounded weird to me. I looked it up in the dictionary and saw that it is the UK form of the word. And because it is the americans who pay... ServicesCatalogue -> ServiceCatalog. I like it more... and saves some bytes of data, it's more ecologic.

**Comment by ignapas** [comment]
> @ignapas Is report issue button always going to be visible? I mean, what happens when you have the inputs bar? Isn't it going to be over the list items?? Wouldn't it be better to put it where it does not interfere e.g. next to the user button?

@pcrespov Your suggestion was implemented and ready to review

**Pull request merged**